ITR/data/base_providers.py

```diff
@@ -107,6 +107,12 @@ def __init__(self, EI_benchmarks: IEIBenchmarkScopes,
         self._EI_benchmarks = EI_benchmarks
         self.temp_config = tempscore_config
         self.column_config = column_config
+        # Benchmark's scope, for which we calculate
```

Comment:
The changes I made in template-v2-scopes change the paradigm from "what is scope you are trying to calculate, I'll give it to you" to "here are all the scope calculations relevant to the benchmark".  In this way, the ITR code doesn't have to guess how the benchmark should be interpreted, it just gives all the answers and the caller decides how to aggregate the scopes.  In this bit of code there's an internal logic prioritizing scopes S1+S2 over S3 if the first case is available.  But that logic may not be correct for all cases (or cases where S1+S2+S3 should be computed).  Best to let the caller aggregate (as the benchmark intends).